src/main/scala/uk/gov/nationalarchives/notifications/messages/Messages.scala

```diff
@@ -30,7 +30,7 @@ trait Messages[T <: IncomingEvent, TContext] {
 
   def sns(incomingEvent: T, context: TContext): Option[SnsMessageDetails] = None
 
-  def govUkNotifyEmail(incomingEvent: T, context: TContext): Option[GovUKEmailDetails] = None
+  def govUkNotifyEmail(incomingEvent: T, context: TContext): List[Option[GovUKEmailDetails]] = Nil
```

Comment:
Can we just change this signature to a `List` (empty or Nil if we don't want to send anything)?. Feels a bit cleaner- `List[Option[X]]`s can always get a bit confusing.